app/policies/comment_policy.rb

```diff
@@ -0,0 +1,28 @@
+class CommentPolicy
+  attr_reader :user, :comment
+
+  def initialize(user, comment)
+    @user = user
+    @comment = comment
+  end
+
+  def show?
+    true
+  end
+  
+  def create?
+    true
+  end
+
+  def update?
+    user == comment.author
+  end
+  
+  def edit?
+    user == comment.author
```

Comment:
For future reference, you could also just call `update?` so that you don't have to repeat rules that are similar.

Reply:
do you mean?
```ruby
  def edit?
    update?
  end
  ```